srcalc/src/main/webapp/css/srcalc.css

```diff
@@ -250,6 +250,16 @@ form.attributeEditForm td.attributeName {
 	width: 20%;
 }
 
+/* Alternate colors on variables within a variable group. 
+   The variable group title should not be included in the alternation.*/
+form.attributeEditForm tbody tr:nth-child(2n+2) {
+    background: #FFF;
+}
+
+form.attributeEditForm tbody tr:nth-child(2n+3) {
```

Comment:
I think I get why you used `2n+3` but a bit more explanation in the comments would help.